### ISSUE TITLE
Create codefmt usage screencast for README.md

### ISSUE THREAD

**Issue by dbarnett** [google/vim-codefmt#25]
Lots of vim plugins have fancy screencasts (animated GIFs) in the README.md showcasing plugin features. The codefmt plugin has several useful features that would make a nice screencast.

We should create a screencast and embed it in our README.md. If we could also include a keyboard monitor in the screencast, that would help show off all the handy mappings.

**Comment by aranmohyeddin** [comment]
Actually I can't even find what the mappings are!

**Comment by dbarnett** [comment]
They're mentioned in the helpfile (specifically `:help codefmt-mappings`). Any clarifications you're looking for there?

**Comment by aranmohyeddin** [comment]
No. I just thought it would have been better if you put a few of them in the readme.md here.
